EIPS/eip-3541.md

```diff
@@ -0,0 +1,50 @@
+---
+eip: 3541
+title: Reject new contracts starting with the 0xEF byte
```

Comment:
```suggestion
title: Contract Leading Byte Reservation
```

Reply:
I'm open to changes, but would like to get feedback from @chfast @gumb0 @AlexeyAkhunov @holiman @chriseth which one they prefer. However I would not make this a blocker for merging, as we can update it afterwards, i.e. in the one marking it review?

Reply:
My personal view is that perhaps the current one is better.

Reply:
I also prefer a more explicit title.